css-viewport/Overview.bs

```diff
@@ -526,6 +526,39 @@ The <dfn export>unscaled</dfn> value of a CSS length relative to an element is t
 	The {{Window/devicePixelRatio}} of a frame is multiplied by the [=effective zoom=] inherited by its parent frame.
 </h3>
 
+<h2 id=extensions-to-the-window-interface>Extensions to the {{Window}} Interface</h2>
+
+<pre class=idl>
+partial interface Window {
+[SameObject, Replaceable] readonly attribute Viewport? viewport;
```

Comment:
Is there a resolution to add this?

Reply:
This was the suggested path by the resolution to add `segments` here rather than `window.visualViewport` as it was originally proposed. @bramus do you agree?

Reply:
The resolution speaks of a "new layout viewport tbd". You're naming it viewport instead of layoutViewport - why? There is already a visualViewport on Window.

Reply:
I don't have a strong opinion on the name. I know that @bramus had some thoughts on adding more properties beside `segments` so I'm not sure what's the best way to name it. @bramus @fantasai : any thoughts?

Reply:
When specs speak of "the viewport" they mean the Layout Viewport, so `viewport` seems fine.